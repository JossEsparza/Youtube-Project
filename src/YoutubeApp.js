import React from 'react';
import { useState } from 'react';
import Navbar from './components/navbar/Navbar';
import { YoutubeAppRouter } from './router/YoutubeAppRouter';
import ToggleContext from './state/ToggleContext';

const YoutubeApp = () => {
  const [currentToggle, setCurrentToggle] = useState(false);
  const [isEnabled, setIsEnabled] = useState(false);
  return (
    <div className={isEnabled ? 'dark' : 'light'}>
      <ToggleContext.Provider
        value={{ currentToggle, setCurrentToggle, isEnabled, setIsEnabled }}
      >
        <Navbar />
        <YoutubeAppRouter />
      </ToggleContext.Provider>
    </div>
  );
};

export default YoutubeApp;
