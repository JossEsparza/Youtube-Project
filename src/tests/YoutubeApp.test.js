import { shallow } from 'enzyme';
import YoutubeApp from '../YoutubeApp';

describe('Tests the YoutubeApp', () => {
  test('should be displayed correctly', () => {
    const wrapper = shallow(<YoutubeApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
