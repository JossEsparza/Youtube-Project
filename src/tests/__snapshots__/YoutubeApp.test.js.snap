// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests the YoutubeApp should be displayed correctly 1`] = `ShallowWrapper {}`;
